gitlab_test.go

```diff
@@ -0,0 +1,115 @@
+package gitlab
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+var (
+// mux is the HTTP request multiplexer used with the test server.
+	mux *http.ServeMux
```

Comment:
Why is this a global var? I don't think `mux` is used anywhere outside `setup()` right? So I would not make this a separate var.